### PR TITLE
update the ci docs to use the pro image

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Comment by MarcelStranak** [comment]
Can we also mention that we now offer `localstack/localstack-pro:2.0.2-bigdata`?
They should know that they can use this image in the CI environment to make their lives easier.

**Comment by HarshCasper** [comment]
> Can we also mention that we now offer `localstack/localstack-pro:2.0.2-bigdata`? They should know that they can use this image in the CI environment to make their lives easier.

I think that's a special use-case that we can document elsewhere.

File: content/en/user-guide/ci/_index.md [hunk 1]
```diff
@@ -33,4 +33,8 @@ The Pro subscription is mainly intended for use on individual user machines. We
...
+The steps required for the integration differ slightly depending on your preferred CI platform. Please refer to the relevant sections below for detailed instructions on the integration process. To follow the instructions, you can first retrieve the CI key from the [Account settings page](https://app.localstack.cloud/account/apikeys) in the [LocalStack Web app](https://app.localstack.cloud).
+
+## CI images
+
+LocalStack Docker images can be used in your CI environment by adding a CI Key. The images are available on [Docker Hub](https://hub.docker.com/r/localstack/localstack/tags), and comprehensive documentation is available on our [Docker images](https://docs.localstack.cloud/references/docker-images/) documentation. Community users can use the `localstack/local stack image, while Pro and Team users can use the `localstack/localstack-pro` image. For Big Data jobs that require services such as EMR, Athena, and Glue, we provide a mono-container that uses the `localstack/localstack-pro:2.0.2-bigdata` image, which bakes in the required dependencies, such as Hadoop, Hive, Presto, into the LocalStack image.
```
Comment (steffyP) [inline content/en/user-guide/ci/_index.md:40]:
```suggestion
LocalStack Docker images can be used in your CI environment by adding a CI Key. The images are available on [Docker Hub](https://hub.docker.com/r/localstack/localstack/tags), and comprehensive documentation is available on our [Docker images](https://docs.localstack.cloud/references/docker-images/) documentation. Community users can use the `localstack/localstack` image, while Pro and Team users can use the `localstack/localstack-pro` image. For Big Data jobs that require services such as EMR, Athena, and Glue, we provide a mono-container that uses the `localstack/localstack-pro:2.0.2-bigdata` image, which bakes in the required dependencies, such as Hadoop, Hive, Presto, into the LocalStack image.
```

File: content/en/user-guide/ci/gitlab-ci/index.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ test:
     - aws s3 ls s3://test/ --endpoint-url ${LOCALSTACK_URL}
 ```
 
-## Activate LocalStack Pro
+## Configuring an API key
 
-You can easily enable LocalStack Pro by adding your API key to the repository's environment variables. Go to your project's **Settings > CI/CD**  and expand the  **Variables**  section. Select the **Add Variable** button and fill in the necessary details. After you create a variable, you can use it in the `.gitlab-ci.yml` file.
+You can easily enable LocalStack Pro by by using the `localstack/localstack-pro` image and adding your API key to the repository's environment variables. Go to your project's **Settings > CI/CD**  and expand the  **Variables**  section. Select the **Add Variable** button and fill in the necessary details. After you create a variable, you can use it in the `.gitlab-ci.yml` file.
```
Comment (steffyP) [inline content/en/user-guide/ci/gitlab-ci/index.md:54]:
```suggestion
You can easily enable LocalStack Pro by using the `localstack/localstack-pro` image and adding your API key to the repository's environment variables. Go to your project's **Settings > CI/CD**  and expand the  **Variables**  section. Select the **Add Variable** button and fill in the necessary details. After you create a variable, you can use it in the `.gitlab-ci.yml` file.
```

**Review by steffyP — APPROVED** [review]
LGTM 👍 

Just a general remark: I think we lately agreed on a new convention: to write only one sentence per line in markdown files. 
Not sure if this is something you already want to do here - maybe you could do it at least for the parts, where you still need to do changes? 🙂 

One a side note, personally I would also prefer that the 
> Here is an example:

sentences would go into a new paragraph, as it makes the reading-flow and comprehension easier.

**Pull request merged**